src/main/java/uk/gov/hmcts/ccd/infrastructure/user/UserAuthorisation.java

```diff
@@ -18,6 +22,15 @@ public AccessLevel getAccessLevel() {
         return accessLevel;
     }
 
+    public Boolean hasJurisdictionRole(String jurisdictionId) {
+        final String jurisdictionRole = "caseworker-" + jurisdictionId;
```

Comment:
Is this always going to be for caseworker? Is it worth to parametrize the method further?

Reply:
Good question. Yes, it is always to be caseworker.

My thinking is that at this point the format of base jurisdiction role (`caseworker-{jid}`) has become a core concept of this project, and is also hard-coded in other places of this project (see `authorizedRolesExtractor()`) , and other projects as well (Document Store, API Gateway). 

Because of that, a different pattern could not be used without refactoring of at least 3 projects and a migration of roles for our IDAM users.

So, should it be parameterised? I wouldn't think so, that would give the expectation it can easily be changed when in fact it can't.

Reply:
Ok fair enough.